examples/bri-3/prisma/schema.prisma

```diff
@@ -18,6 +18,11 @@ model BpiSubject {
   publicKey   String
 }
 
+model BpiAccount {
+  id    String @id @default(uuid())
+  nonce String
```

Comment:
@ognjenkurtic @skosito @Manik-Jain @Kasshern @Ybittan ... we need to agree on the nonce typing. `int` is easiest,  or `string` is ok as well to give more flexibility. for BRI-3 I would recommend sticking with `int` for simplicity's sake ... this would allow for example synchronization with e.g. an Ethereum account if desired.

Reply:
I agree with `int` too.

Also, if we want to be future safe and handle any large numbers, we can also consider incorporating number or bigInt

Reply:
it was int initially but comment was to move to string here https://github.com/eea-oasis/baseline/pull/507#discussion_r956461635
i will revert to int